app/views/admin/moderation/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_for petition, url: admin_petition_moderation_path(petition), method: :patch do |f| -%>
+  <%= form_row do %>
+    <%= label_tag :moderation_approve, nil do %>
```

Comment:
GDS tend to wrap radio and checkbox groups in a `<fieldset class="inline">` but that can be done later.